tests/testsuite.src/run_extensions.at

```diff
@@ -124,6 +124,77 @@ AT_CHECK([$COBCRUN_DIRECT ./prog], [0],
 AT_CLEANUP
 
 
+AT_SETUP([GCOS literals with EBCDIC symbols])
+AT_KEYWORDS([extensions gcos ebcdic])
+
+AT_DATA([prog.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      prog.
+       DATA             DIVISION.
+       WORKING-STORAGE  SECTION.
+       01 X             PIC X(11).
+       01 T             PIC 9.
+       01 S             USAGE COMP-1 VALUE ZERO.
+       PROCEDURE        DIVISION.
+           MOVE 1 TO T
+           MOVE ""135,151,151"bar"195,194,234"" TO X
+           IF X NOT EQUAL "foobarBAZ" PERFORM SHOW.
+
+           MOVE 2 TO T
+           MOVE "  "135,151,151"bar"195 194"Z" TO X
+           IF X NOT EQUAL "  foobarBAZ" PERFORM SHOW.
+
+           MOVE 3 TO T
+           MOVE ' '135 151,151'bar'195,194'Z' TO X
+           IF X NOT EQUAL " foobarBAZ" PERFORM SHOW.
+
+           *> GCOS7-specific: should give a ! in between square brackets:
+           *> note square brackets need to be doubled due to m4 preprocessing.
```

Comment:
Can you double the [[ ]] of the prog.cob content instead? Possibly much easier: replace [] by () / {} ?

Reply:
done using [[...]]